galleries/examples/images_contours_and_fields/image_exact_placement.py

```diff
@@ -0,0 +1,127 @@
+"""
+=========================================
+Placing images, preserving relative sizes
+=========================================
+
+By default Matplotlib resamples images created with `~.Axes.imshow` to
+fit inside the parent ~.Axes`.  This can mean that images that have very
+different original sizes can appear similar in size.
+
+Sometimes, however,  it is desirable to keep the mages the same relative size, or
+even to make the images keep exactly the same pixels as the original data.
+Matplotlib does not automatically make either of these things happen,
+but it is possible with some manual manipulation.
+
+Preserving relative sizes
+=========================
+
+If the size of the images are amenable, we can preserve the relative sizes of two
+images by using either the *width_ratio* or *height_ratio* of the subplots.  Which
+one you use depends on the shape of the image and the size of the figure.
+
+By default the two images are made a similar size, despite one being 1.5 times the width
+of the other:
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+import matplotlib.patches as mpatches
+
+# make the data:
+N = 450
+x = np.arange(N) / N
+y = np.arange(N) / N
+
+X, Y = np.meshgrid(x, y)
+R = np.sqrt(X**2 + Y**2)
+f0 = 5
+k = 100
+a = np.sin(np.pi * 2 * (f0 * R + k * R**2 / 2))
+A = a[:100, :300]
+B = A[:40, :200]
+
+# plot with default axes handling:
+fig, axs = plt.subplots(1, 2, facecolor='aliceblue')
+
+axs[0].imshow(A, vmin=-1, vmax=1)
+axs[1].imshow(B, vmin=-1, vmax=1)
+rect = mpatches.Rectangle((0, 0), 200, 40, linewidth=1, edgecolor='r', facecolor='none')
+axs[0].add_patch(rect)
+
+# %%
+# Note that both images are rendered at a 1:1 ratio, but are made to look almost the
+# same width, despite image B being smaller than image A.
+#
+# We can control the relative sizes using the *width_ratios* argument _if_ the images
+# are wider than they are tall and shown side by side, as is the case here.
+
+fig, axs = plt.subplots(1, 2, width_ratios=[300/200, 1], facecolor='aliceblue')
+
+axs[0].imshow(A, vmin=-1, vmax=1)
+rect = mpatches.Rectangle((0, 0), 200, 40, linewidth=1, edgecolor='r', facecolor='none')
+axs[0].add_patch(rect)
+axs[1].imshow(B, vmin=-1, vmax=1)
+
+# %%
+# Given that the data subsample is in the upper left of the larger image,
+# it might make sense if the top of the smaller Axes aligned with the top of the larger.
+# This can be done manually by using `~.Axes.set_anchor`, and using "NW" (for
+# northwest).
+
+fig, axs = plt.subplots(1, 2, width_ratios=[300/200, 1], facecolor='aliceblue')
+
+axs[0].imshow(A, vmin=-1, vmax=1)
+rect = mpatches.Rectangle((0, 0), 200, 40, linewidth=1, edgecolor='r', facecolor='none')
+axs[0].add_patch(rect)
+axs[0].set_anchor('NW')
+axs[1].imshow(B, vmin=-1, vmax=1)
+axs[1].set_anchor('NW')
+
+# %%
+# Note that this procedure still leaves large white spaces (that can be trimmed in a
+# final product by ``bbox_inches="tight"`), and is not very general.  For instance, if
```

Comment:
```suggestion
# final product by ``bbox_inches="tight"``), and is not very general.  For instance, if
```
I also wonder if it’s worth making this a link to `savefig` so that it’s _really_ obvious what you’re referring to.